src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCostOneDay = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const minDaysForDiscount = 3;
+  const maxDaysForMaxDiscount = 7;
+
+  if (days < minDaysForDiscount) {
+    return rentalCostOneDay * days;
+  }
+
+  if (days >= minDaysForDiscount && days < maxDaysForMaxDiscount) {
+    return rentalCostOneDay * days - minDiscount;
+  }
+
+  if (days >= maxDaysForMaxDiscount) {
+    return rentalCostOneDay * days - maxDiscount;
+  }
```

Comment:
Last if is useless. It will always be `true` with given conditions.

```suggestion
  return rentalCostOneDay * days - maxDiscount;
```